qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -816,6 +843,21 @@ def _connect_signals(self):
         view.renderProcessTerminated.connect(
             self._on_render_process_terminated)
         view.iconChanged.connect(self.icon_changed)
+        #WORKAROUND for https://bugreports.qt.io/browse/QTBUG-65223
+        if QT_VERSION_STR.startswith('5.10'):
+            page.loadProgress.connect(self._on_load_progress_fake)
+            self.loadProgressFake.connect(self._on_load_progress)
```

Comment:
This indirection seems kind of confusing to me - why not simply connect `page.loadProgress` to `self._on_load_progress` unconditionally, and then additionally connect it to `_on_load_progress_fake` (which will then only emit `loadFinishedFake`)?